### PR TITLE
[5.5] Fix the extra ')'

### PR DESCRIPTION
**Pull request by devilkiller-ag**
This PR is for solving issue #1453 
# Changes made
Changed the link to trace on Wikipedia in section [Trace and Positivity Conditions](https://qiskit.org/textbook/ch-quantum-hardware/density-matrix.html#Trace-and-Positivity-Conditions-) to eliminate extra ')'. 

# Justification
There is an extra ')' after the link to Wikipedia which needs to be eliminated. It was due to '(' and ')' in the link to Wikipedia. I encoded them with '%28' for '(' and '%29' for '('.

### CODE REVIEW

**Comment by devilkiller-ag** [comment]
Kindly review my changes @frankharkins @HwajungKang @vabarbosa @kUmezawa @kifumi @nkanazawa1989

**Comment by frankharkins** [comment]
@devilkiller-ag could you instead use an HTML link (as described in [this comment](https://github.com/qiskit-community/qiskit-textbook/issues/1453#issuecomment-1240713793)). The reason is I'm not sure how this will render on the live site, whereas I know the HTML links work correctly.

File: content/ch-quantum-hardware/density-matrix.ipynb [hunk 1]
```diff
@@ -3143,7 +3143,7 @@
     "\n",
     "1. Its trace must be equal to one: \n",
     "\n",
-    "    Remembering that the [trace of a matrix](https://en.wikipedia.org/wiki/Trace_(linear_algebra)) (denoted as $\\text{Tr}$) is the sum of its diagonal terms, we have:\n",
+    "    Remembering that the [trace of a matrix](https://en.wikipedia.org/wiki/Trace_%28linear_algebra%29) (denoted as $\\text{Tr}$) is the sum of its diagonal terms, we have:\n",
```
Comment (frankharkins) [inline content/ch-quantum-hardware/density-matrix.ipynb:3146]:
```suggestion
    "    Remembering that the <a href=\"https://en.wikipedia.org/wiki/Trace_(linear_algebra)\">trace of a matrix</a> (denoted as $\\text{Tr}$) is the sum of its diagonal terms, we have:\n",
```

**Comment by devilkiller-ag** [comment]
Sure I will do this today

**Comment by frankharkins** [comment]
@devilkiller-ag you can click the "commit suggestion" button here to do it within GitHub: https://github.com/qiskit-community/qiskit-textbook/pull/1503#discussion_r1015289294

**Comment by devilkiller-ag** [comment]
@frankharkins Thanks for the help. I committed your suggestion. I think we are good to go now.

**Review by frankharkins — APPROVED** [review]
Thank you!

**Pull request merged**